--- a/libcxx/include/CMakeLists.txt
+++ b/libcxx/include/CMakeLists.txt
@@ -1731,7 +1731,7 @@
 # Make sure to map the generated include directory back to libc++'s actual source directory when generating
 # debug information. Otherwise, the debug information will refer to generated headers which are created during
 # the build and generally not persistent.
-target_add_compile_flags_if_supported(cxx-headers INTERFACE "-fdebug-prefix-map=${LIBCXX_GENERATED_INCLUDE_DIR}=${LIBCXX_SOURCE_DIR}/include")
+target_add_compile_flags_if_supported(cxx-headers INTERFACE "-fdebug-prefix-map=${LIBCXX_GENERATED_INCLUDE_DIR}=/usr/src/noj-wasi-v2/libcxx/include")
 
 if (LIBCXX_INSTALL_HEADERS)
   foreach(file ${files})
